src/libs/SelectionScraper/index.js

```diff
@@ -81,9 +80,6 @@ const getHTMLOfSelection = () => {
         }
     }
 
-    // Remove any element that isn't selectable to prevent copying unnecessary text/items
-    div.querySelectorAll(`div[style*="user-select: ${styles.userSelectNone.WebkitUserSelect}"]`).forEach(item => item.remove());
```

Comment:
Yikes. This does not look like React Native to me 😅

Reply:
@marcaaron There are a couple of of other places where DOM-related methods are used inside `SelectionScraper` such as `div.appendChild()` and `closest()`, and this is what inspired my solution.

https://github.com/Expensify/App/blob/c7a8a880b1985b81ef26ebe9119c4c33c827694c/src/libs/SelectionScraper/index.js#L79

https://github.com/Expensify/App/blob/c7a8a880b1985b81ef26ebe9119c4c33c827694c/src/libs/SelectionScraper/index.js#L24

I'm the [original PR](https://github.com/Expensify/App/pull/18228)'s author, and I'm currently investigating this unintended behavior.